MsCorePkg/Library/SecureBootKeyStoreLib/SecureBootKeyStoreLib.c

```diff
@@ -0,0 +1,99 @@
+/** @file PlatformKeyLib.c
+
+  Copyright (C) Microsoft Corporation. All rights reserved.
+  SPDX-License-Identifier: BSD-2-Clause-Patent
+
+**/
+
+#include <Uefi.h>
+#include <UefiSecureBoot.h>
+#include <Guid/ImageAuthentication.h>
+#include <Library/SecureBootVariableLib.h>
+
+#include <Library/DebugLib.h>
+#include <Library/MemoryAllocationLib.h>
+#include <Library/PcdLib.h>
+
+#define PLATFORM_SECURE_BOOT_KEY_COUNT  2
+
+SECURE_BOOT_PAYLOAD_INFO  *gSecureBootPayload     = NULL;
+UINT8                     gSecureBootPayloadCount = 0;

```

Comment:
Why are these prefixed with `g`? They're local to the module just like `m` prefixed ones.

Also, why are they needed? They just contain information already available in the other globals.

Reply:
This is from the previous version of SecureBootKeyStoreLib that was in mu_oem_sample. I did not remove it when it became unnecessary. I will be removing in the next commit. Thanks!